frappe/public/js/frappe/list/list_view.js

```diff
@@ -320,9 +320,9 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 
 	setup_freeze_area() {
 		this.$freeze = $(
-			`<div class="freeze flex justify-center align-center text-muted">${__(
-				"Loading"
-			)}...</div>`
+			`<div class="freeze flex justify-center align-center text-muted">
+				${__("Loading...", null, "Freeze area in list view")}
```

Comment:
are there different translations for Loading and Loading...
?

Reply:
In general, translators have to know the whole string that will be displayed, not just an arbitrary part of it. But we can revert this change for now to not break existing translations.